wiki/Tournaments/TWC/2022/en.md

```diff
@@ -197,6 +213,36 @@ The osu!taiko World Cup 2022 is run by various community members.
 
 ## Match results
 
+### Round of 16
+
+Detailed statistics for this round can be found [here](https://docs.google.com/spreadsheets/d/19vfkfIEnt0dhqC2cXuCvDSwhvFAeSbw5i_ZAw2l1CtA/edit?rm=minimal).
+
+Saturday, March 26, 2022
+
+| Team A | | | Team B | Match link |
```

Comment:
ci apparently doesn't catch this, but empty cells should have two spaces, not one

Reply:
there seem to be a few tournament articles with this issue, so i'll fix them in a separate pr